bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -6,6 +6,130 @@
+        </span>
+        <span class="mzp-c-button-icon-text">{{ ftl('footer-refresh-donate') }}</span>
+      </a>
+      <div class="mzp-c-footer-language">
+        {% include 'includes/protocol/lang-switcher-refresh.html' %}
+       </div>
+    </div>
+    <nav class="mzp-c-footer-secondary">
+      <div class="mzp-c-footer-legal">
+        <p class="mzp-c-footer-license" rel="license">
+          {% set moco_link = 'href="%s" data-link-type="footer" data-link-text="Mozilla Corporation"'|safe|format(url('mozorg.home')) %}
+          {% set mofo_link = 'href="https://foundation.mozilla.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer" rel="external noopener" data-link-type="footer" data-link-text="Mozilla Foundation"'|safe %}
+          {{ ftl('footer-refresh-visit-mozilla-corporations', moco_link=moco_link, mofo_link=mofo_link) }}<br>
+          {{ ftl('footer-refresh-portions-of-this-content', url=url('foundation.licensing.website-content'), current_year=current_year|string) }}
+        </p>
+        <ul class="mzp-c-footer-terms">
+          <li><a href="{{ url('privacy.notices.websites') }}" data-link-type="footer" data-link-text="Privacy">{{ ftl('footer-refresh-websites-privacy-notice') }}</a></li>
+          <li>
+            {# Link to /privacy/websites/cookie-settings/ is a legal requirement and should not be removed. It must be present on every page (Issue 14213). #}
+            <a href="{{ url('privacy.cookie-settings') }}" data-link-type="footer" data-link-text="Cookies">{{ ftl('footer-refresh-websites-cookies') }}</a>
+          </li>
+          <li><a href="{{ url('legal.index') }}" data-link-type="footer" data-link-text="Legal">{{ ftl('footer-refresh-websites-legal') }}</a></li>
+          <li><a href="{{ url('mozorg.about.governance.policies.participation') }}" data-link-type="footer" data-link-text="Community Participation Guidelines">{{ ftl('footer-refresh-community-participation-guidelines') }}</a></li>
+          {% if ftl_has_messages('footer-about-this-site') %}
+            <li><a href="{{ url('mozorg.about.this-site') }}" data-link-type="footer" data-link-text="About this site">{{ ftl('footer-refresh-about-this-site') }}</a></li>
+          {% endif %}
+        </ul>
+      </div>
+    </nav>
+    <img class="c-footer-logo-image-bottom" src="{{ static('img/logos/mozilla/footer-refresh/mozilla.png') }}" alt="mozilla" width="1376" height="285">
+  </div>
+</footer>
+{% else %}
```

Comment:
**question:** might it be worth splitting the new / old footer into different files/includes to keep things a bit more separated?

Reply:
I agree it is worth splitting the new/old footer. I kept it in one file for now since it would make testing easier. I think the intention is to see the new footer/navigation with existing pages, not just the new ones to decide whether or not to roll out the footer globally.